riscv/csrs.cc

```diff
@@ -957,12 +957,6 @@ bool envcfg_csr_t::unlogged_write(const reg_t val) noexcept {
   return masked_csr_t::unlogged_write(adjusted_val);
 }
 
-// implement class henvcfg_csr_t
-henvcfg_csr_t::henvcfg_csr_t(processor_t* const proc, const reg_t addr, const reg_t mask, const reg_t init, csr_t_p menvcfg):
-  envcfg_csr_t(proc, addr, mask, init),
-  menvcfg(menvcfg) {
-}
-
```

Comment:
Its unclear why this change/restructuring was needed.

Reply:
I just only want to put member functions with the same class together.

Reply:
Since that restructuring/refactoring is unrelated to this PR, I suggest that this PR not do this. The refactoring/restructuring of code should be in a separate PR.